wire-ios/Wire-iOS/Sources/UserInterface/SelfProfile/SelfProfileViewController.swift

```diff
@@ -227,7 +229,7 @@ extension SelfProfileViewController: SettingsPropertyFactoryDelegate {
         }
 
         guard newValue else {
-            try? settingsPropertyFactory.userSession?.appLockController.deletePasscode()
+            try? settingsPropertyFactory.userSession?.deleteAppLockPasscode()
```

Comment:
do we need to go through the settingsPropertyFactory ? we have it now directly, no?
```suggestion
            try? userSession.deleteAppLockPasscode()
```